### DISCUSSION TITLE
constrained electrodes selection in optimisation

### DISCUSSION THREAD

**Discussion by ggrrll** [simnibs/simnibs#148] [8. Simulation workflows]
We were wondering if there is a way (how difficult would be) to implement (hard) constraints in the selection of electrodes, during the optimisation.
In particular, we need to respect a min distance among them — this is a question that arises also for instance if one uses your largest positioning system (SPM12)

thanks

**Comment by TorgeW** [comment]
This is not a feature of SimNIBS at the moment but might be considered in the future.
